### ISSUE TITLE
Video is not streaming while the client device is outside of the home network

### ISSUE THREAD

**Issue by bogdanbun** [Brandawg93/homebridge-nest-cam2#23]
I noticed that while connecting from outside of home network I still get the thumbnail, but the video is not streaming. The client device is trying to connect for some time and then gives up. It looks like the ffmgep process is actually starting on the hombridge side, but for some reason it is not getting to the client device:

```
  410 ?        Ssl    9:11 homebridge                                          
 6294 ?        S      0:01  \_ ffmpeg -use_wallclock_as_timestamps 1 -i - -c:v libx264 -an -pix_fmt yuv420p -preset ultrafast -tune zerolatency -payload_type 99 -ssrc 2086046 -f rtp -srtp_out_suite AES_CM_128_HMAC_SHA1_80 -srtp_out_par...

```
If the client device is located in the home network it works as expected. At the same time, other cameras (with native HomeKit support) work just fine in either case.
Is it just my experience?

**Comment by Brandawg93** [comment]
I actually had the same issue and was able to fix it by reinstalling homebridge including deleting the persist and accessories folders. If that doesn’t work, check and make sure there isn’t a firewall issue. There should be more info in logs if you run homebridge in debug mode. Also note that nest has been having connectivity issues for almost a week now.

**Comment by Brandawg93** [comment]
Try using the new release, v1.1.8.

**Comment by bogdanbun** [comment]
Removing the content of the "persists" folder (which entailed the accessory reinstallation in the HomeKit) did the trick. Thank you.
I should mention I tried installing v1.1.8 but it was failing to even generate the thumbnail. So I rolled back to v1.1.7.

**Comment by Brandawg93** [comment]
v1.1.9 fixes v1.1.8. Good to hear its working for you.

**Comment by bogdanbun** [comment]
I'm having some problems installing the version 1.1.9. I will start another issue for that.
